rollup.config.js

```diff
@@ -10,26 +10,42 @@ const packageJson = require('./package.json');
 const peerDependencies = Object.keys(packageJson.peerDependencies);
 const dependencies = Object.keys(packageJson.dependencies);
 
+function globals() {
```

Comment:
Now looks like we have two places defining "globals": here, and `umdFullConfig.globals`. Let's try to only use one.